DiTails/LevelDataService.cs

```diff
@@ -49,13 +50,13 @@ internal async Task<Beatmap> Vote(Beatmap beatmap, bool upvote, CancellationToke
                 {
                     steam = true;
                 }
-                else if (!(_platformUserModel is OculusPlatformUserModel))
+                else
```

Comment:
This needs testing on the Oculus version of the game. 
I only have the Steam version, and the only two implementations of `IPlatformUserModel` are `MockPlatformUserModel` and `SteamPlatformUserModel`

Not sure if the `PlatformUserModel.dll` is different on two platforms

Reply:
nvm, pepega me 
just found the platform field